XenModel/FriendlyNames.resx

```diff
@@ -2006,4 +2006,4 @@
   <data name="VM.TemplateDescription-Other install media" xml:space="preserve">
     <value>If the operating system you plan to use is not listed, you may be able to install it by selecting this template.  We recommend that only advanced users attempt to use this template, and you should note that our products have been tested running only the supported distributions and specific versions covered by the standard supplied templates.</value>
   </data>
-</root>
+</root>
```

Comment:
Very trivial, but since you'll be making other changes, you can also revert this file, this change is not needed.